contracts/PlotClarificationManager.sol

```diff
@@ -53,6 +53,8 @@ contract PlotClarificationManager is AbstractApplication {
     address applicant;
     bytes32 ledgerIdentifier;
     uint256 spaceTokenId;
+    
+    uint256 fee;
```

Comment:
@Jonybang Why do we need this extra variable to be stored in the state?

Reply:
Because submitApplication was reworked because of too many variables error